lib/Sema/CSSimplify.cpp

```diff
@@ -11115,7 +11115,7 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyMemberConstraint(
 
       auto instanceTy = baseObjTy->getMetatypeInstanceType();
 
-      auto impact = 2;
+      auto impact = 10;
```

Comment:
How about we increase this to 4 and lower `RemoveInvalidCall` impact to 3, we already increase this by a lot if a the base doesn't have a member.

Reply:
Sure. Testing that.